addons/web/static/src/scss/primary_variables.scss

```diff
@@ -260,11 +260,11 @@ $o-btns-bs-override: map-merge((
     "secondary": (
         background: $o-gray-300,
         border: $o-gray-300,
-        color: $o-gray-800,
+        color: $o-main-text-color,
```

Comment:
Here you defined `$o-main-text-color` for secondary and light buttons. [In Enterprise](https://github.com/odoo/enterprise/pull/85027/files#diff-bc677ed1f61394a870c165a237c7966f14f5a4d3290f31e0448406cbf6477389R76) you defined `$text-900`. I'm not sure to understand why, did I missed something ? 🤔 

I'm wondering if using `$o-main-text-color` for buttons is the right way to go.
Let' assume we want to change the body color to a dark red, for example.
This means that light and secondary buttons will be impacted, which is not what we want, right ?

So I guess using `$text-900` would be the way to go to keep the right contrast without being impacted to changes coming from the body color, what do you think ?

Reply:
Yes you're right! I've been hesitant and it's a leftover of my overthinking. `900` makes more sense as buttons should be independent from body-color, I agree.